independent-projects/arc/processor/src/main/java/io/quarkus/arc/processor/Methods.java

```diff
@@ -186,6 +186,13 @@ public MethodVisitor visitMethod(int access, String name, String descriptor, Str
                         classLevelBindings, bytecodeTransformerConsumer, transformUnproxyableClasses));
             }
         }
+        for (DotName i : classInfo.interfaceNames()) {
+            ClassInfo interfaceInfo = getClassByName(beanDeployment.getIndex(), i);
+            if (interfaceInfo != null) {
+                finalMethodsFoundAndNotChanged.addAll(addInterceptedMethodCandidates(beanDeployment, interfaceInfo, candidates,
```

Comment:
Interface methods cannot be final.
It should be enough to call `addInterceptedMethodCandidates()` here.